modules/influxdb/influxdb_test.go

```diff
@@ -7,6 +7,7 @@ import (
 	"testing"
 	"time"
 
+	influxdb2 "github.com/influxdata/influxdb-client-go/v2"
```

Comment:
suggestion: for consistency with the v1 import
```suggestion
	influxclient2 "github.com/influxdata/influxdb-client-go/v2"
```